src/main/resources/assets/tconstruct/book/puny_smelting/en_us/upgrades/ranged/freezing.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:freezing",
+  "text": [
+    {
+      "text": "Elsa may be jealous."
+    }
+  ],
+  "effects": [
+    "Fired projectiles slows enemies",
+	"Multiple levels, each level slows enemies for longer",
```

Comment:
Its not just slow, it also does more damage

It does do more damage to specific enemies like blazes, but that might not be worth saying